Public/Src/Cache/ContentStore/Library/Sessions/ContentSessionBase.cs

```diff
@@ -211,7 +211,16 @@ protected ContentSessionBase(string name)
                     Tracer,
                     () => PutFileCoreAsync(operationContext, hashType, path, realizationMode, urgencyHint, _counters[ContentSessionBaseCounters.PutFileRetries]),
                     extraStartMessage: $"({path},{realizationMode},{hashType}) trusted=false",
-                    extraEndMessage: _ => "trusted=false",
+                    extraEndMessage: (result) => {

```

Comment:
Formatting issue.
The open brace should be on the new line if the lambda is not a single line one.